tools/retest/main.go

```diff
@@ -162,7 +168,9 @@ func jobsToRetestFromComments(userComments, allComments []string) ([]string, err
 	missingTests := make([]string, 0, len(jobsToRetest))
 	for job, times := range jobsToRetest {
 		toTest := times - testedJobs[job]
+		log.Printf("Job %q has been tested already %d out of %d times", job, testedJobs[job], times)
 		if toTest < 1 {
+			log.Printf("Exceeded number of retests for %q", job)
```

Comment:
Let's cut on noise and only log in the interesting case:
```suggestion
		if toTest < 1 {
			log.Printf("Exceeded max number (%d) of retests for %q: %d", times, job, testedJobs[job])
```